packit/security.py

```diff
@@ -95,6 +101,8 @@ def download_gpg_key_if_needed(self, key_fingerprint: str) -> None:
         except Exception as ex:
             raise PackitException(f"Cannot receive a gpg key: {key_fingerprint}", ex)
 
+        raise PackitException(f"Cannot receive a gpg key: {key_fingerprint}")
```

Comment:
Couldn't these 3 exceptions be merged into one? It seems that if there is no `return` above, one of them will be called anyways. Or do I misunderstand something?

Reply:
I've removed one of them, no idea how to squash the remaining two.

Reply:
Yes, it seems this is all that can be done. Thanks!